app/bundles/EmailBundle/Form/Type/EmailType.php

```diff
@@ -159,6 +161,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ]
         );
 
+        $builder->add(
+            'useOwnerAsMailer',
+            'yesno_button_group',
+            [
+                'label'      => 'mautic.email.use.owner.as.mailer',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => [
+                    'class'   => 'form-control',
+                    'tooltip' => 'mautic.email.use.owner.as.mailer.tooltip',
+                ],
+                'data'     => (bool) empty($options['data']->getUseOwnerAsMailer()) ? $this->configOwnerAsMailerSetting : $options['data']->getUseOwnerAsMailer(),
```

Comment:
I think you meant:
```
'data'     => (bool) (empty($options['data']->getUseOwnerAsMailer()) ? $this->configOwnerAsMailerSetting : $options['data']->getUseOwnerAsMailer()),
```

Reply:
I think this is for discustion. 
If I use global owner as mailer from, then I cannot disable owner  on email settings (allways is true). 
Then we should hide it or allow disable owner as mail for each email even If global settings exist. 
Global settings should be as default for new email, but on edit, we should hide or allow edit this option

Reply:
Thats fine, what I meant:

`var_dump((bool) empty($empty) ? 0 : 1);` returns `int(0)`
`var_dump((bool) (empty($empty) ? 0 : 1));` returns `bool(false)`

Not a big deal, just noticing you might not have intended this.